client/src/components/listStatusItem/listStatusItem.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon, ListView } from 'patternfly-react';
+
+import { SimpleTooltip } from '../simpleTooltIp/simpleTooltip';
+
+const ListStatusItem = ({
+  id,
+  count,
+  emptyText,
+  tipSingular,
+  tipPlural,
+  expanded,
+  expandType,
+  toggleExpand,
+  iconType,
+  iconName
+}) => {
+  let renderExpandContent = (iconType, iconName, count, text) => {
+    if (iconType && iconName) {
+      return (
+        <React.Fragment>
+          <Icon className="list-view-compound-item-icon" type={iconType} name={iconName} />
+          <strong>{count}</strong>
+        </React.Fragment>
+      );
+    } else {
+      return (
+        <span>
+          <strong>{count}</strong>
+          {` ${text}`}
+        </span>
+      );
+    }
+  };
+
+  if (count > 0) {
+    return (
+      <ListView.InfoItem className="list-view-info-item-icon-count">
+        <SimpleTooltip id={`${id}_tip`} tooltip={`${count}  ${count === 1 ? tipSingular : tipPlural}`}>
```

Comment:
Optional: the id string/template literals, looks like the id is always supposed to be a string, we worried about someone jamming a number in there? if so we could just flip ```tip_${id}```

Reply:
The propType is string. I could make it accept either, but if you pass an number you do get a console warning but it works OK.